src/main/java/com/avispl/dal/communicator/polycom/videoos/PolycomVideoOS.java

```diff
@@ -1470,6 +1497,9 @@ private void retrieveCommunicationProtocolsInfo(Map<String, String> statistics)
      */
     private void retrieveApplications(Map<String, String> statistics) throws Exception {
         JsonNode response = doGet(APPS, JsonNode.class);
+        if (response == null) {
+            return;
+        }

```

Comment:
similar to above, we should debug log that response was null.

There are more locations below that we should debug log the fact that we could not perform action due to null response